src/AssetsTransferApi.spec.ts

```diff
@@ -157,13 +167,181 @@ describe('AssetTransferAPI', () => {
 			});
 		});
 	});
+
+	describe('fetchCallType', () => {
+		describe('RelayToSystem', () => {
+			it('Should correctly return Teleport', () => {
+				const assetCallType = relayAssetsApi['fetchCallType'](
+					'0',
+					'1000',
+					['ksm'],
+					Direction.RelayToSystem,
+					AssetType.Native,
+					false,
+					relayAssetsApi.registry
+				);
+
+				expect(assetCallType).toEqual('Teleport');
+			});
+		});
+		describe('RelayToPara', () => {
+			it('Should correctly return Reserve', () => {
+				const assetCallType = relayAssetsApi['fetchCallType'](
+					'0',
+					'2023',
+					['ksm'],
+					Direction.RelayToPara,
+					AssetType.Native,
+					false,
+					relayAssetsApi.registry
+				);
+
+				expect(assetCallType).toEqual('Reserve');
+			});
+		});
+		describe('SystemToRelay', () => {
+			it('Should correctly return Teleport', () => {
+				const assetCallType = systemAssetsApi['fetchCallType'](
+					'1000',
+					'0',
+					['ksm'],
+					Direction.SystemToRelay,
+					AssetType.Native,
+					false,
+					systemAssetsApi.registry
+				);
+
+				expect(assetCallType).toEqual('Teleport');
+			});
+		});
+		describe('SystemToSystem', () => {
```

Comment:
Web have 2 `describe`s here with `SystemToSystem` I think we can just have one with both `its` under it.